src/palladio/SOPAssign.cpp

```diff
@@ -68,6 +68,103 @@ RuleFileInfoUPtr getRuleFileInfo(const MainAttributes& ma, const ResolveMapSPtr&
 	return rfi;
 }
 
+enum class SpareParmType { FLT, BOOL, STR, FLT_ARRAY, BOOL_ARRAY, STR_ARRAY, FLT_ENUM, STR_ENUM, COLOR };
+using SpareParmTypeMap = std::map<std::wstring, SpareParmType>;
```

Comment:
As this function basically translates the type from Houdini to CGA, renaming to `CGAAttributeType` (or similar) would be clearer.

Reply:
renamed